omod-common/src/main/java/org/openmrs/module/webservices/helper/TaskAction.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.helper;
+
+import org.openmrs.scheduler.TaskDefinition;
+
+import java.util.List;
+
+public class TaskAction {
+	
+	public enum Action {
+		SCHEDULETASK, SHUTDOWNTASK, RESCHEDULETASK, RESCHEDULEALLTASK, DELETE, ONSTARTUP, ONSHUTDOWN;
+	}
+	
+	public TaskAction() {
+	}
+	
+	public TaskAction(Action action, List<TaskDefinition> tasks, boolean startAll) {
+		this.tasks = tasks;
+		this.action = action;
+		this.allTasks = startAll;
+	}
+	
+	private List<TaskDefinition> tasks;
+	
+	private Action action;
+	
+	private Boolean allTasks;
```

Comment:
Why do you make this nullable? Isn't more friendly to change the type to boolean and have it default to false?

Reply:
okay I will do it like that way :-)

Reply:
anyway I want to check again `isAllTasks == null` to change the boolean value from false to true?

Reply:
How can it be null when the type is changed to boolean?

Reply:
There will be no null type here anymore.
Default value or the Boolean will be False So I will check for that here.